### PR TITLE
Create t3ii-essential_supplies_gloves_usage.md

### PR DESCRIPTION
**Pull request by The-Kergan**
@Nelviticus or @geoffwasham files need to be uploaded for this page please:

Dropbox\COVID-19\1-WORK OUTPUT\WEEK 3\FINAL\T3ii_use_of_gloves_v3.pdf
Dropbox\COVID-19\5-Images\Infographics\gloves_safe_removal.jpg

Thanks.

### CODE REVIEW

File: docs/tools/t3ii-essential_supplies_gloves_usage.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+#### When Leaving the House - No
+
+**In summary:** it is not necessary to wear gloves when out in public. Make sure you wash your hands regularly and do not touch your face. 
+
+**Medical Gloves – No.** There is no strong evidence that recommends that you should wear medical gloves when leaving the house or while traveling on public transport.
+
+Medical gloves are helpful in reducing transmission of pathogens like COVID-19 in healthcare workers, but at the same time they do not provide complete protection. 
+
+When leaving the house, it is fundamental to wash your hands regularly and / or use alcohol gel solution at the first available opportunity. Using gloves is unnecessary and can also lead to an increased the risk of germ transmission, if not used correctly.
```
Comment (Nelviticus) [inline docs/tools/t3ii-essential_supplies_gloves_usage.md:30]: 'an increased (the) risk' - lose 'the' or change to 'an increase in the risk'

File: docs/tools/t3ii-essential_supplies_gloves_usage.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+**__After using gloves__**
+
+**Medical Gloves:** remove them carefully and wash hands thoroughly with soap and water before wearing a new pair. 
+
+**Reusable gloves/rubber gloves:** remove them carefully and clean them with soap, water and bleach (20 ml / 1 teaspoon of bleach to 1 L / 2 pints of water). Allow to dry. Wash hands thoroughly with soap and water. 
+
+**Source:** [BCCDC](http://www.bccdc.ca/health-professionals/clinical-resources/covid-19-care), [HSE](http://www.hse.gov.uk/)
+
+### Putting-on and removing gloves safely
```
Comment (Nelviticus) [inline docs/tools/t3ii-essential_supplies_gloves_usage.md:53]: 'Putting on' (no hyphen)

File: docs/tools/t3ii-essential_supplies_gloves_usage.md (hunk 1 above)
Comment (The-Kergan) [inline docs/tools/t3ii-essential_supplies_gloves_usage.md:30]:
```suggestion
When leaving the house, it is fundamental to wash your hands regularly and / or use alcohol gel solution at the first available opportunity. Using gloves is unnecessary and can also lead to an increased risk of germ transmission, if not used correctly.
```

File: docs/tools/t3ii-essential_supplies_gloves_usage.md (hunk 2 above)
Comment (The-Kergan) [inline docs/tools/t3ii-essential_supplies_gloves_usage.md:53]:
```suggestion
### Putting on and removing gloves safely
```

**Comment by The-Kergan** [comment]
Typos taken care of.

**Pull request merged**